integration/graphql-types.ts

```diff
@@ -1,3 +1,5 @@
 export const possibleTypes = {
   HasName: ["Author", "Book"],
 };
+
+export type HasNameTypes = Author | Book;
```

Comment:
I presume that `Author` and `Book` type will already be imported or exist in this file already? Thoughts?

Reply:
Yep, they would be defined in this file by other plugins in the pipeline.